### ISSUE TITLE
Technical Readout text in TOE disappears randomly

### ISSUE THREAD

**Issue by JonhyBurner** [MegaMek/mekhq#2225]
Mekhq 47.13
Windows 10
Java 1.8.0 ver 271

When selecting "unit" tab in the TOE view, sometimes the Technical Readout text is blank. If that happens, it stays that way, apparently as long as the session continues. If I load a previous save, previously missing text might appear, or not. It seems rather random, and if I click around a bit, all units end up lacking the text.

**Comment by sixlettervariables** [comment]
We'll need your mekhqlog.txt as this is likely caused by an unhanded exception.

Also please attach your campaign and any customs you use in a ZIP file.

**Comment by JonhyBurner** [comment]
Sorry, I wanted to upload them before, but apparently Firefox and Github sometimes don't mingle very well.

[megameklog.zip](https://github.com/MegaMek/mekhq/files/5530034/megameklog.zip)
[mekhqlog.zip](https://github.com/MegaMek/mekhq/files/5530038/mekhqlog.zip)
[Campaign save.gz](https://github.com/MegaMek/mekhq/files/5530084/Campaign.save.gz)


Here's how it looks, by the way:
![Missing TRO](https://user-images.githubusercontent.com/67933202/98931338-cbc12980-24d5-11eb-8879-3164481f4c55.jpg)

**Comment by sixlettervariables** [comment]
@Windchild292 
```
Loading atlas: data\images\units\mechs\mechs_atlas.png
Loading atlas: data\images\units\DamageDecals\DamageDecals_atlas.png
Loading atlas: data\images\units\mechs\mechs_atlas.png
Loading atlas: data\images\units\mechs\mechs_atlas.png
Loading atlas: data\images\units\mechs\mechs_atlas.png
Loading atlas: data\images\units\mechs\mechs_atlas.png
sun.awt.image.PNGImageDecoder$PNGException: Broken file
	at sun.awt.image.PNGImageDecoder.pngassert(Unknown Source)
	at sun.awt.image.PNGImageDecoder.handleChunk(Unknown Source)
	at sun.awt.image.PNGImageDecoder.getData(Unknown Source)
	at sun.awt.image.PNGImageDecoder.produceImage(Unknown Source)
	at sun.awt.image.InputStreamImageSource.doFetch(Unknown Source)
	at sun.awt.image.ImageFetcher.fetchloop(Unknown Source)
	at sun.awt.image.ImageFetcher.run(Unknown Source)
sun.awt.image.PNGImageDecoder$PNGException: Broken file
	at sun.awt.image.PNGImageDecoder.pngassert(Unknown Source)
	at sun.awt.image.PNGImageDecoder.handleChunk(Unknown Source)
	at sun.awt.image.PNGImageDecoder.getData(Unknown Source)
	at sun.awt.image.PNGImageDecoder.produceImage(Unknown Source)
	at sun.awt.image.InputStreamImageSource.doFetch(Unknown Source)
	at sun.awt.image.ImageFetcher.fetchloop(Unknown Source)
	at sun.awt.image.ImageFetcher.run(Unknown Source)
Loading atlas: data\images\units\mechs\mechs_atlas.png
sun.awt.image.PNGImageDecoder$PNGException: Broken file
	at sun.awt.image.PNGImageDecoder.pngassert(Unknown Source)
	at sun.awt.image.PNGImageDecoder.handleChunk(Unknown Source)
	at sun.awt.image.PNGImageDecoder.getData(Unknown Source)
	at sun.awt.image.PNGImageDecoder.produceImage(Unknown Source)
	at sun.awt.image.InputStreamImageSource.doFetch(Unknown Source)
	at sun.awt.image.ImageFetcher.fetchloop(Unknown Source)
	at sun.awt.image.ImageFetcher.run(Unknown Source)
sun.awt.image.PNGImageDecoder$PNGException: Broken file
	at sun.awt.image.PNGImageDecoder.pngassert(Unknown Source)
	at sun.awt.image.PNGImageDecoder.handleChunk(Unknown Source)
	at sun.awt.image.PNGImageDecoder.getData(Unknown Source)
	at sun.awt.image.PNGImageDecoder.produceImage(Unknown Source)
	at sun.awt.image.InputStreamImageSource.doFetch(Unknown Source)
	at sun.awt.image.ImageFetcher.fetchloop(Unknown Source)
```

**Comment by Windchild292** [comment]
Fixed in #2221
